implot3d.cpp

```diff
@@ -48,6 +48,7 @@
 #define IMGUI_DEFINE_MATH_OPERATORS
 #endif
 
+#include <cmath>
```

Comment:
Hi @floooh! After the PR #11 `#include<cmath>` is no longer needed :)

```suggestion
```

Reply:
Done (via separate commit).